packages/core/tests/unit/mixins/perfect-scrollbar-test.js

```diff
@@ -0,0 +1,13 @@
+import EmberObject from '@ember/object';
+import { run } from '@ember/runloop';
+import PerfectScrollbarMixin from 'navi-core/mixins/perfect-scrollbar';
+import { module, test } from 'qunit';
+
+module('Unit | Mixin | perfect scrollbar');
+
+test('it works', function(assert) {
```

Comment:
This isn't really testing any code that we've written. It's just checking that `Ember.Object.create()` works so I think this test is unnecessary.

Reply:
I'm testing that destroy runs without crashing here.

Reply:
Could we change the test name to reflect that?